install.sh

```diff
@@ -3973,6 +4013,62 @@ install_into_existing_repositories() {
     return
 }
 
+############################################################
+# Disable a locally installed LFS hook `$1` if detected.
+#   By default, a detected lfs hook is disabled and
+#   backuped.
+#   Sets the variable `DELETE_DETECTED_LFS_HOOKS` for
+#   later invocations.
+#
+# Returns:
+#   0 on moved or deleted, 1 otherwise
+############################################################
+disable_lfs_hook_if_detected() {
+    HOOK_FILE="$1"
+
+    if [ -f "$HOOK_FILE" ] &&
+        grep -q -rE "(git\s+lfs|git-lfs)" "$HOOK_FILE"; then
+
+        # Load the global decision
+        if [ -z "$DELETE_DETECTED_LFS_HOOKS" ]; then
+            DELETE_DETECTED_LFS_HOOKS=$(git config --global githooks.deleteDetectedLFSHooks)
+        fi
+
+        if ! is_non_interactive && [ -z "$DELETE_DETECTED_LFS_HOOKS" ]; then
+            echo "! There is an LFS commmand statement in \`$HOOK_FILE\`."
+            echo "  Githooks will call LFS hooks internally and LFS should not be called twice."
+            printf "  Do you want to delete this hook instead of disabling/backuping it? (No, never, yes, always) [N,n,y,a] "
```

Comment:
Can we do `(Yes, no, all, skip all) [Y/n/a/s]` like elsewhere? (though in this case `No, yes, all, skip all`)
This way, you can accept both lower and uppercase letters.